project-base/storefront/public/locales/cs/common.json

```diff
@@ -201,8 +204,10 @@
     "or immediately in {{ count }} stores_other": "nebo ihned na {{ count }} prodejnách",
     "Order": "Objednávka",
     "Order not found.": "Objednávka nenalezena",
+    "order number": "číslo objednávky",
     "Order number": "Číslo objednávky",
```

Comment:
Isn't it better to have just one translation string and modify the first letter using CSS? Seems better to me...

Reply:
Fixed